index.bs

```diff
@@ -2007,17 +2007,14 @@ Note: The {{CollectedClientData}} may be extended in the future. Therefore it's
         the syntax defined by [[RFC6454]].
 
     :   <dfn>tokenBinding</dfn>
-    ::  This member contains information about the state of the [=Token Binding=] protocol used when communicating with the
-        [=[RP]=].
+    ::  This optional member contains information about the state of the [=Token Binding=] protocol used when communicating
```

Comment:
Please change "optional" to "OPTIONAL" here.

Reply:
Done.